lib/web_ui/lib/src/engine/text/paragraph.dart

```diff
@@ -1745,6 +1745,10 @@ void _applyTextStyleToElement({
   if (color != null) {
     cssStyle.color = colorToCssString(color);
   }
+  final ui.Color? background = style._background?.color;
```

Comment:
Totally unimplemented? Yikes! :P

Reply:
Long story :P

In the old implementation, we used to only support plain text (with very minimal rich text support). If the paragraph has a red background for example, we used to just draw a red rectangle behind the paragraph.

With the recent rich text implementation, we started drawing rectangles for each span that has a background (in canvas2d mode). But in DOM mode, we reused some of the old code, which didn't do backgrounds for spans.

Reply:
Ahhh, yes, I remember the rectangle drawing part!